src/types/pre_checkout_query.rs

```diff
@@ -35,5 +35,5 @@ pub struct PreCheckoutQuery {
     pub shipping_option_id: Option<String>,
 
     /// Order info provided by the user.
-    pub order_info: Option<OrderInfo>,
+    pub order_info: OrderInfo,
```

Comment:
This wouldn't work though, would it?

A test like this will fail:
```rust
let json = r#"{
  "id":"",
  "from":{"id":0,"is_bot":false,"first_name":"F"},
  "currency":"EUR",
  "total_amount":1
}"#;
let _: PreCheckoutQuery = serde_json::from_str(json).unwrap();
```
(no `order_info` field is provided)

One way to fix this is to **not** make this field required. Another way is to make it default to all `None`s:
```suggestion
    #[serde(default)]
    pub order_info: OrderInfo,
```
(this also requires `Default derived on `OrderInfo`)

I'm not sure what we should do, since I've never used the payment API. I'd guess that it isn't useful to distinguish not present `order_info` with present, but all-`None`s `order_info`, right? So we should go with the latter option? What do you think?

If we are going with the latter option, please also apply the same to [`SuccessfulPayment::order_info`](https://docs.rs/teloxide/0.5.3/teloxide/types/struct.SuccessfulPayment.html#structfield.order_info)